cmd/kubeadm/app/preflight/checks_windows.go

```diff
@@ -46,7 +46,7 @@ func (ipuc IsPrivilegedUserCheck) Check() (warnings, errorList []error) {
 		}
 	}
 
-	return nil, []error{errors.New("user is not running as administrator")}
+	return nil, []error{errors.Wrap(err, "user is not running as administrator")}
```

Comment:
this is not valid. only the above 2 errors should be wrapped.

please use rebase and "git push --force" to keep the number of commits to one.

Reply:
good catch. my bad wasn't paying close attention.